coreneuron/io/phase2.hpp

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include "coreneuron/io/nrn_filehandler.hpp"
+#include "coreneuron/io/user_params.hpp"
+
+#include <memory>
+
+namespace coreneuron {
+struct NrnThread;
+struct NrnThreadMembList;
+struct Memb_func;
+struct Memb_list;
+
+class Phase2 {
```

Comment:
IIRC, first we are reading data into Phase1 + Phase2 and then we reserve the vectors in NrnThread and do memcopy. As these data structures take 90% of available memory, we can't use this double copy approach.

Reply:
give me an idea on that.
The only way I see is modifying `nrn2core_*` to something more clean.
Do you agree I start that?